app/MMB/src/scenes/timeline/DaySeparator.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet, Text, Color } from '@kiwicom/mobile-shared';
+import {
+  Translation,
+  DateFormatter,
+  DateUtils,
+} from '@kiwicom/mobile-localization';
+
+import TimelineRow from './TimelineRow';
+
+type Props = {|
+  +date: Date,
+|};
+
+const DaySeparator = (props: Props) => {
+  const { date } = props;
+  let translatedDay = null;
+  if (DateUtils.diffInDays(date, new Date()) === 0) {
+    translatedDay = (
+      <Translation id="mmb.booking_timeline.today" textTransform="uppercase" />
+    );
+  } else {
+    translatedDay = (
+      <Translation
+        passThrough={DateFormatter(new Date(date)).formatToDate()}
+        textTransform="uppercase"
+      />
+    );
+  }
+  return (
+    <TimelineRow
+      rowStyle={styles.container}
+      rightColumn={<Text style={styles.date}>{translatedDay}</Text>}
+    />
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    paddingTop: 10,
+    paddingBottom: 10,
+    backgroundColor: Color.grey.$300,
+  },
+  date: {
+    color: Color.ink.light,
+    fontFamily: 'CircularPro-Book',
+    fontSize: 14,
+    fontWeight: 'bold',
```

Comment:
You should not change the `fontFamily` or `fontSize` manually here. These are default values so just omit it, please...